specs/src/specs/data_structures.md

```diff
@@ -370,46 +369,21 @@ The blob share commitment rules may introduce empty shares that do not belong to
 
-| name                    | type                                            | description                                                                                                                                                                                                                                                                                                                       |
-|-------------------------|-------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `index`                 | `uint64`                                        | Index of this intermediate state root in the list of intermediate state roots. This information is lost when splitting intermediate state roots into fixed-sized [shares](./shares.md), and needs to be re-added here for fraud proof support. Allows linking an intermediate state root to a [transaction](#wrappedtransaction). |
-| `intermediateStateRoot` | [IntermediateStateRoot](#intermediatestateroot) | Intermediate state root. Used for fraud proofs.                                                                                                                                                                                                                                                                                   |
+| name            | type       | description                                                                                                                                            |
+|-----------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tx`            | `bytes`    | Actual transaction.                                                                                                                                    |
+| `share_indexes` | `[]uint32` | Share indexes (in row-major order) of the first share for each blob this transaction pays for. Needed for light verification of proper blob inclusion. |
+| `type_id`       | `string`   | Type ID of the IndexWrapper transaction type. Always set to `"INDX"`                                                                                   |
 
-#### IntermediateStateRoot
-
-| name   | type                      | description                                                                              |
-|--------|---------------------------|------------------------------------------------------------------------------------------|
-| `root` | [HashDigest](#hashdigest) | Root of intermediate state, which is composed of the global state and the validator set. |
+### PayForBlobData
```

Comment:
Shouldn't the `IndexWrapper` go beneath the pay for blob data?

Reply:
2652b1021f8af9b81383699f20664aa6d34401c7